adoc/MAIN.release-notes.adoc

```diff
@@ -1,5 +1,5 @@
-Release Notes: SUSE Cloud Application Platform 1.3
-==================================================
+Release Notes: SUSE Cloud Application Platform 1.3.1
+====================================================
 
```

Comment:
I still can't build adoc files :( So, comparing to the published version: https://www.suse.com/releasenotes/x86_64/SUSE-CAP/1/ In my experience we referenced the major release number in release notes titles, e.g. "Release Notes
SUSE Cloud Application Platform 1.3", and other comparable locations, and then referenced the minor release numbers in sections in the content.